data/card_catalogue.json

```diff
@@ -1198,17 +1160,17 @@
         {
           "title": "Sprint",
           "upgrade": "Teleport",
-          "tier": "blue",
+          "tier": "green",
           "details": {
-            "rules": "Until end of combat, you can travel 2 extra squares with each movement",
+            "rules": "Until end of combat, you can travel twice as fast",
```

Comment:
This is a bit of a trick- since we don't actually have a consistent system of movement `twice as fast` is up to the DM to simulate.